pkg/ee/kubelb/resources/seed-cluster/deployment.go

```diff
@@ -137,7 +137,7 @@ func DeploymentReconcilerWithoutInitWrapper(data kubeLBData) reconciling.NamedDe
 				{
 					Name:    resources.KubeLBDeploymentName,
 					Image:   repository + ":" + imageTag,
-					Command: []string{"/usr/local/bin/ccm"},
+					Command: []string{"/ccm"},
```

Comment:
This looks weird. Are we sure this shouldn't be `ccm` only? Unless the image changed and there was a symlink in it?

Reply:
The image changed

Reply:
But the path is now `/ccm`.

Reply:
And what's the issue with that now?

Reply:
`Command: []string{"/ccm"},` should be the right value for this line of code. Anyways all of this being release tested right now, if there is some issue I'll come back to this. But I don't really understand what you are on here atm

Reply:
`ccm` would be resolved in the PATH, so `ccm` and `/usr/local/bin/ccm` are equivalent and I can understand the change.

Changing it to `/ccm` is different. Try running `/ls` on your laptop.

I just wanted to make sure you didn't leave the `/` there by accident.